src/memray/reporters/templates/flamegraph.html

```diff
@@ -16,6 +16,11 @@
           title="Hide CPython eval frames and other, memray-related frames" checked>
     <label class="form-check-label text-white bg-dark">Hide Non-Relevant Frames</label>
 </div>
+<div class="form-check mr-3">
+  <input class="form-check-input" type="checkbox" data-toggle="tooltip" id="hideImportSystem"
+          title="Hide frames related to the Python import system" >
+    <label class="form-check-label text-white bg-dark">Hide import system Frames</label>
```

Comment:
The other checkbox is "Hide Non-Relevant Frames", so this should be
```suggestion
    <label class="form-check-label text-white bg-dark">Hide Import System Frames</label>
```

Either both should be title case or neither should.